raftcpp.vcxproj.filters

```diff
@@ -19,4 +19,9 @@
       <Filter>源文件</Filter>
     </ClCompile>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This file shouldn't be committed since it's a local config file, and not all users need this.

As a cpp project, a `Cmakelist.txt` is enough.

Reply:
We can add this files into `.gitignore`.